pkg/components/discover/watcher_proc_test.go

```diff
@@ -145,7 +145,7 @@ func TestProcessNotReady(t *testing.T) {
 		},
 	}
 
-	procs, err := acc.listProcesses(true)
+	procs, err := acc.listProcesses(true, 0)
```

Comment:
Can you please add a test in this file that ensures that young processes are not found?